packages/react/src/FormControl/FormControl.tsx

```diff
@@ -172,17 +172,15 @@ const FormControl = React.forwardRef<HTMLDivElement, FormControlProps>(
                 {slots.leadingVisual}
               </Box>
             )}
-            {!slots.label?.props.visuallyHidden || slots.caption ? (
```

Comment:
is there a reason we don't need this check anymore?

Reply:
I couldn't figure out why needed this check, and removing it doesn't break any VRTs or unit tests.

Reply:
Did some local testing and couldn't force a difference either. Went down a bit of a rabbit hole and saw you [authored](https://github.com/primer/react/pull/1836/files#diff-868bfd75f5bcb508e151876612d32055aa262949ee9c675eed2187efc311ec08R164) this change so I'll trust you, approving! Let me know if you want help running integration checks or testing in RL